dashboard/src/app/(authenticated)/(diagram)/components/Modals/UserTaskRun.tsx

```diff
@@ -0,0 +1,161 @@
+import { FC, useRef } from 'react'
+import { UserTaskModal } from '@/app/(authenticated)/(diagram)/context'
+import { useModal } from '@/app/(authenticated)/(diagram)/hooks/useModal'
+import { Dialog, DialogPanel, DialogTitle } from '@headlessui/react'
+import { DialogBody } from 'next/dist/client/components/react-dev-overlay/internal/components/Dialog'
+import { XMarkIcon } from '@heroicons/react/24/outline'
+import { getVariable, getVariableValue } from '@/app/utils'
+import { UserTaskRunStatus } from 'littlehorse-client/dist/proto/user_tasks'
+
+export const UserTaskRun: FC<UserTaskModal> = ({ data, nodeRun, userTaskNode }) => {
+  const { showModal, setShowModal } = useModal()
+  const refDiv = useRef(null)
```

Comment:
I wonder why is this necessary if TaskRun doesn't need it?

Reply:
I was needing certain properties like nodeRun.endTime and the cancellationException which are not part of the UserTask object itself.